examples/tests/sdk_tests/util/spawn.test.w

```diff
@@ -0,0 +1,82 @@
+bring util;
+bring expect;
+
+let assertThrows = inflight (expected: str, block: (): void) => {
+  let var error = false;
+  try {
+    block();
+    } catch actual {
+      expect.equal(actual, expected);
+      error = true;
+    }
+    expect.equal(error, true);
+};
+
+// spawn() in preflight doesn't work
+let program = "echo";
+let args = ["undefined"];
+
+let child = util.spawn(program, args);
```

Comment:
interesting, doesn't it throw?

Reply:
No, but the child doesn't get spawned, so everything is `undefined`.

Reply:
I think we would need to wait for #5425 to be merged then.

Reply:
it will take a while, please change it to try/catch / assertThrow 
so it will pass the PR actions and we can merge it :) 
![image](https://github.com/winglang/wing/assets/39455181/156c6fb8-d6ea-4c58-8d26-799fa110a8af)